code/AssetLib/LWS/LWSLoader.cpp

```diff
@@ -888,6 +891,7 @@ void LWSImporter::InternReadFile(const std::string &pFile, aiScene *pScene, IOSy
     if (anims.size()) {
         master->mAnimations = new aiAnimation *[master->mNumAnimations = 1];
         aiAnimation *anim = master->mAnimations[0] = new aiAnimation();
+        master->mAnimations[0] = nullptr;
```

Comment:
This one is definitely wrong, it forgets the pointer that was just create above. Prefer one assignment per line to avoid such confusing code.

Reply:
Right.